### PR TITLE
New package: armagetronad-0.2.8.3.3

### PR DESCRIPTION
**Pull request by Vaelatern**
Different templates only because the configure script can not configure both dedicated server roles and a game client at the same time.

### CODE REVIEW

**Comment by Vaelatern** [comment]
Need to add `config_files`

File: srcpkgs/armagetronad/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'armagetronad'
+pkgname=armagetronad
+version=0.2.8.3.3
+revision=1
+build_style=gnu-configure
+conf_files="/etc/games/${pkgname}/*.cfg
+		/etc/games/${pkgname}/*.srv"
```
Comment (xtraeme) [inline srcpkgs/armagetronad/template]: both package handle the same configuration files or are they different?

File: srcpkgs/armagetronad/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+configure_args="--enable-master --enable-main --enable-music"
+hostmakedepends="m4"
+makedepends="libX11-devel libxml2-devel libICE-devel SDL-devel libpng-devel
+		libjpeg-turbo-devel SDL_image-devel SDL_mixer-devel libGL glu-devel"
```
Comment (xtraeme) [inline srcpkgs/armagetronad/template]: `libGL` is unnecessary, as well as `SDL-devel`.

File: srcpkgs/armagetronad-dedicated/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+version=0.2.8.3.3
+revision=1
+_name=armagetronad
+wrksrc="${_name}-${version}"
+build_style=gnu-configure
+conf_files="/etc/games/${pkgname}/*.cfg
+		/etc/games/${pkgname}/*.srv"
+configure_args="--enable-dedicated"
+hostmakedepends="m4"
+makedepends="libX11-devel libxml2-devel libICE-devel SDL-devel libpng-devel
```
Comment (xtraeme) [inline srcpkgs/armagetronad-dedicated/template]: ditto

File: srcpkgs/armagetronad/template (hunk 1 above)
Comment (Vaelatern) [inline srcpkgs/armagetronad/template]: Both packages handle the same configuration, but they install it to `/etc/games/${pgkname}`

Comment (Vaelatern) [inline srcpkgs/armagetronad/template]: I'm adding a patch to put the configuration files in the same place.

Comment (Vaelatern) [inline srcpkgs/armagetronad/template]: @xtraeme, same config, but similar enough files in both packages that they'd collide otherwise.

**Comment by Vaelatern** [comment]
Now both packages build out of the same template and source.

File: srcpkgs/armagetronad/template (hunk 1 above)
Comment (Vaelatern) [inline srcpkgs/armagetronad/template]: Should I change the `conf_files` variable in the subpackage?

**Pull request merged**